### DISCUSSION TITLE
Issue with workflow on Linux with OSM Data

### DISCUSSION THREAD

**Discussion by Denis-Vannier** [orbisgis/geoclimate#938] [Q&A]
Hi !

I have an issue when running the workflow on Linux with the OSM data. It happened with a particular city, Nantes (but not with the neighboring town, Rezé), and I can't figure out what could be wrong in that case. 

It gave me this alert : "Table "INPUT_SEA_LAND_MASK__BDAE592F_F077_4E4B_9B1D_B9A4764B4BC1" not found".

Should I add something in the config file to avoid this ?

Thanks !

(PS : the workflow run fine with BD Topo data)

![Capture d’écran 2024-02-22 à 21 04 28](https://github.com/orbisgis/geoclimate/assets/9679672/4cffe885-7965-4873-abd3-95240bd787c2)
[Hist_Terminal_Geoclimate_OSM_Nantes.txt](https://github.com/orbisgis/geoclimate/files/14379216/Hist_Terminal_Geoclimate_OSM_Nantes.txt)

**Comment by ebocher** [comment]
@Denis-Vannier  thanks to report this issue. 
It seems fixed on the current snaphot version avalaible here https://github.com/orbisgis/geoclimate/wiki/Download
Please test and ping us if the same bug occurs.

**Reply by Denis-Vannier** [reply]
Thanks,
I installed the current snapshot version tonight, but found the same bug.
Here's the log and the config file. 
[config_file_osm.json](https://github.com/orbisgis/geoclimate/files/14379663/config_file_osm.json)
[Résultat de Terminal.txt](https://github.com/orbisgis/geoclimate/files/14379665/Resultat.de.Terminal.txt)

**Comment by ebocher** [comment]
Investigation in progress...

**Comment by Denis-Vannier** [comment]
The new snaphot version fixed this issue. Thanks !
